src/test/ui/parser/byte-literals.rs

```diff
@@ -8,5 +8,5 @@ pub fn main() {
     b'	';  //~ ERROR byte constant must be escaped
     b''';  //~ ERROR byte constant must be escaped
     b'é';  //~ ERROR byte constant must be ASCII
-    b'a  //~ ERROR unterminated byte constant
+    b'a  //~ ERROR E0763
```

Comment:
```suggestion
    b'a  //~ ERROR [E0763]: unterminated character literal
```
Could you avoid removing the human readable annotation?